src/Notes/WC_Admin_Notes_Onboarding_Payments.php

```diff
@@ -0,0 +1,79 @@
+<?php
+/**
+ * WooCommerce Admin: Payments reminder note.
+ *
+ * Adds a notes to complete the payment methods.
+ *
+ * @package WooCommerce Admin
+ */
+
+namespace Automattic\WooCommerce\Admin\Notes;
+
+defined( 'ABSPATH' ) || exit;
+
+use \Automattic\WooCommerce\Admin\Features\Onboarding;
+
+/**
+ * WC_Admin_Notes_Onboarding_Payments.
+ */
+class WC_Admin_Notes_Onboarding_Payments {
+	const NOTE_NAME = 'wc-admin-onboarding-payments-reminder';
+
+	/**
+	 * Creates a note to remind store owners to set up payments.
+	 */
+	public static function possibly_add_onboarding_payments_note() {
+		// This note should only be added if the task list is still shown.
+		if ( ! Onboarding::should_show_tasks() ) {
+			return;
+		}
+
+		// Make sure payments task was skipped at least 3 days ago.
+		$three_days = 3 * DAY_IN_SECONDS;
+		( ( time() - $wc_admin_installed ) >= $seconds );
```

Comment:
@joshuatf Should this be `$three_days` instead of `$seconds` ?

Reply:
Thanks for catching this, @rrennick.  I goofed by leaving this line in.  Removed in https://github.com/woocommerce/woocommerce-admin/pull/4157/commits/4f0f7cdede1bdc1eb6db725f68781880a97247e3 and also renamed to `$three_days_in_seconds` for clarity.